httpserver/http_connection.cpp

```diff
@@ -35,17 +35,17 @@ namespace http = boost::beast::http; // from <boost/beast/http.hpp>
 
 /// +===========================================
 
-static constexpr auto LOKI_FILE_SERVER_TARGET_HEADER =
-    "X-Loki-File-Server-Target";
-static constexpr auto LOKI_FILE_SERVER_VERB_HEADER = "X-Loki-File-Server-Verb";
-static constexpr auto LOKI_FILE_SERVER_HEADERS_HEADER =
-    "X-Loki-File-Server-Headers";
+static constexpr auto OXEN_FILE_SERVER_TARGET_HEADER =
```

Comment:
I was going to say that this needs to be coordinated with Session clients, but it looks like they are not even used there anymore (along with other proxy/non-onion stuff). I could remove proxy code in a separate PR.